ImGui.uplugin

```diff
@@ -18,6 +18,11 @@
 			"Name": "ImGui",
 			"Type": "DeveloperTool",
 			"LoadingPhase": "PreDefault"
+		},
+		{
+			"Name": "ImPlotWidgets",
+			"Type": "Runtime",
```

Comment:
Something I forgot to go over with Modules is that they can have different 'types'.
Usually Runtime, Developer or Editor. I'd make this one the same Type as the ImGui one above
```suggestion
			"Type": "DeveloperTool",
```